spring-cloud-kubernetes-commons/src/test/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapConfigPropertiesTests.java

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright 2013-2021 the original author or authors.
```

Comment:
from here and down there are only tests. I have tried to add java docs that would clearly explain what I am testing. I really wanted to add some integration tests too, as in my public repo - I did add quite a few for this. Unfortunately, I am still unclear of far too many things about that integration tests module. When I will wrap my head around it better, I will probably add them. I hope that is OK for now.